Runtime/Scripts/GltFast.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2020-2021 Andreas Atteneder
```

Comment:
Please avoid/remove non-changes like this. They screw up the blame-history of a document.

Reply:
Will do! My bad.